blog/post1.md

```diff
@@ -0,0 +1,96 @@
+# Jaeger CLickhouse
+This is an implementation of Jaeger's [storage plugin](https://github.com/jaegertracing/jaeger/tree/master/plugin/storage/grpc) for ClickHouse.
+See as well [jaegertracing/jaeger/issues/1438](https://github.com/jaegertracing/jaeger/issues/1438) for historical discussion regarding Clickhouse plugin.
+
+## Project status
```

Comment:
it seems that the blog post copies a lot of from the repository's main readme file